src/app/shared/models/mentor.model.ts

```diff
@@ -0,0 +1,9 @@
+import { DataType } from './data-type.model';
+
+export interface Mentor extends DataType {
+  name: string;
+  detail: string;
+  photoUrl: string;
+  profession: string;
+  placeOfBirth: string;
```

Comment:
If possible, add two more atributes: "order" and "role".
Order will be to order the mentors in the page. The one with order 0 goes first, the one with order 1 goes after that, and so on.
Role will be to define if that person is a mentor or a facilitator or any other thing. It would be amazing if you have something like: roleEn and roleSp for the translation.